### ISSUE TITLE
Value of stopDebuggerOnAppExit in Launch.json is ignored

### ISSUE THREAD

**Issue by jimallison53** [rokucommunity/roku-debug#159]
The plugin ignores the value of "stopDebuggerOnAppExit ". Proper syntax of "stopDebuggerOnAppExit " in Launch.json requires a value, e.g. 

"stopDebuggerOnAppExit " : true    or "stopDebuggerOnAppExit " : false

If the key is specified with a value of "false", the "false" is ignored and the debugger still exits upon app shutdown.

The plugin should use the *value* of "stopDebuggerOnAppExit ", rather than the mere presence of the key.

**Comment by TwitchBronBron** [comment]
Are you using `enableDebugProtocol: true`? If so, then we have no way to keep the debugger active. Roku shuts it down on their end. 

If not, then perhaps this is indeed a bug.

**Comment by TwitchBronBron** [comment]
Yeah we should probably update the description of that property to explain this better.
